.ci/scripts/install-tools.sh

```diff
@@ -4,4 +4,3 @@ set -exuo pipefail
 .ci/scripts/install-go.sh
```

Comment:
I think we can rid of this line

Reply:
If `go` is installed and matching the version it will skip the installation, so, we can remove it and see how it goes